views/package.html.twig

```diff
@@ -30,79 +30,76 @@
         <p class="lead">{{ package.highest.description }}</p>
         {% endif %}
 
-        <table class="table table-striped">
+        {% if package.highest.homepage %}
+        <div class="row">
+            <div class="col-lg-2 text-right"><strong>Homepage</strong></div>
+            <div class="col-lg-10"><a href="{{ package.highest.homepage }}">{{ package.highest.homepage }}</a></div>
```

Comment:
Using ``col-lg-*`` without having classes for the lower resolutions creates a weird thing, because ``.row`` has a negative horizontal margin to compensate the grid gutter of columns, but your divs don't create these gutters.

If you always want this on 2 columns, use ``col-xs-*`` (grids are bottom-up in bootstrap. ``col-xs`` will apply to higher resolutions as long as you don't overwrite it). Otherwise use ``col-xs-12``. And in any case, it can use 2 columns before reaching ``-lg-``